src/views/assets/Platform/PlatformDetail/index.vue

```diff
@@ -72,7 +72,3 @@ export default {
   }
 }
 </script>
-
-<style scoped>
-
-</style>
```

Comment:
The provided code contains no errors, irregularities, or optimization suggestions. It adheres to best practices and appears to have been well-written for use in modern TypeScript environments.